gradle.properties

```diff
@@ -21,4 +21,4 @@ android.useAndroidX=true
 android.enableJetifier=true
 
 # TODO resolve `./gradlew test` failing when the following isn't set
-android.enableUnitTestBinaryResources=false
\ No newline at end of file
+android.enableUnitTestBinaryResources=true
```

Comment:
nit: The comment above is now no longer valid or at least misleading?

Reply:
Removed